advocate_europe/assets/scss/components/_herounit.scss

```diff
@@ -13,7 +13,7 @@
 
     @media only screen and (max-width: $screen-xs) {
         min-height: 200px;
-        @include rem(padding, 40px 0);
+        padding: rem(40px) 0;
```

Comment:
I am confused by all the styling in the hero unit class. Why is that line even needed?